src/app/static/src/components/parameters/EditPhases.vue

```diff
@@ -2,27 +2,32 @@
   <div>
     <modal class="phase-modal" :open="open">
       <h3>Edit {{paramGroup && paramGroup.label}}</h3>
-      <p>Click on a Phase to drag it to a new start date.</p>
-        <div class="phase-editor" @mouseup="mouseUp">
-          <div class="phases-container">
-            <div ref="rail" class="rail">
-              <div v-for="(value, index) in sliderValues" :id="`slider-${index}`"
-                   :key="index"
-                   role="slider"
-                   :tabindex="index"
-                   :style="{ left:`${sliderPosition(value)}%`, zIndex: value.value.zIndex }"
-                   class="slider"
-                   :class="phaseClassFromIndex(index+1)"
-                   :aria-valuemin="sliderMin(index)"
-                   :aria-valuenow="value.value.daysFromStart"
-                   :aria-valuetext="displayPhases[index].start"
-                   :aria-valuemax="sliderMax(index)"
-                   :aria-label="`Phase ${displayPhases[index].index}`"
-                   @mousemove.stop.prevent="mouseMove(index, $event)"
-                   @mousedown.stop.prevent="mouseDown(index, $event)"
-                   @mouseup="mouseUp">
-                <div class="slider-spike" :class="phaseClassFromIndex(index+1)"></div>
-                <div class="slider-text">
+      <p>
+        Click on a Phase to drag it to a new start date.
+        Rt values must be between {{rtMin}} and {{rtMax}}.
+      </p>
+      <div class="phase-editor" @mouseup="mouseUp">
+        <div class="phases-container">
+          <div ref="rail" class="rail">
+            <div v-for="(value, index) in sliderValues"
+                 :id="`slider-${index}-${sliderUpdateKeys[index].value}`"
+                 :key="index"
+                 role="slider"
+                 :tabindex="index"
+                 :style="{ left:`${sliderPosition(value)}%`, zIndex: value.value.zIndex }"
+                 class="slider"
+                 :class="phaseClassFromIndex(index+1)"
+                 :aria-valuemin="sliderMin(index)"
+                 :aria-valuenow="value.value.daysFromStart"
+                 :aria-valuetext="displayPhases[index].start"
+                 :aria-valuemax="sliderMax(index)"
+                 :aria-label="`Phase ${displayPhases[index].index}`"
+                 @mousemove="mouseMove(index, $event)"
+                 @mousedown="mouseDown(index, $event)"
+                 @mouseup="mouseUp">
+              <div class="slider-spike" :class="phaseClassFromIndex(index+1)"></div>
+              <div class="slider-text">
+                <div @mousedown.prevent="">
```

Comment:
This prevents unintended and distracting selection of text when dragging phases...